HISTORY.md

```diff
@@ -12,6 +12,7 @@
 
 * Support for AWS S3 "AssumeRole" temporary credentials [#1882](https://github.com/TileDB-Inc/TileDB/pull/1882)
 * Added support for Hilbert order sorting for sparse arrays. [#1880](https://github.com/TileDB-Inc/TileDB/pull/1880)
+* Added support for an in-memory backend used with bootstrap option "--enable-memfs" [#1873](https://github.com/TileDB-Inc/TileDB/pull/1873)
```

Comment:
Let's add the word "experimental" somewhere in here until we have benchmarked performance and integrated with all of the other unit tests. Maybe: "Added experimental support ..."